### ISSUE TITLE
it seems that bosh-bootstrap uses micro bosh cli and not bosh-init

### ISSUE THREAD

**Issue by ramonskie** [cloudfoundry-community/bosh-bootstrap#319]
i noticed that bosh micro cli is going to deprecated soon
and that bosh-init should be used

are there plans to do this with the bosh boot strap to?

**Comment by drnic** [comment]
Yes bosh-init is the future and when I start using it I'll move bosh-bootstrap to use it.




But currently bosh-init is not a good new user experience - it is slow to use; so I don't recommend it for new users.
